src/content.rs

```diff
@@ -66,6 +66,9 @@ pub struct Head {
     /// This provides the path info the file
     /// If empty, the path info is filled in using the request url
     pub path_info: Option<String>,
+    /// This provides the path from which to dynamicaly load content for the body
+    /// If empty, the body of the current file is used.
+    pub dynamic_body_source: Option<String>,
```

Comment:
The examples mention `dynamic_content_source` - is there some mapping going on or are is that a glitch with the examples?  It might be useful to have a unit test or an example in the repo that is definitive on the matter.

(I don't have an opinion though `body` makes it clearer that it is strictly a body-for-body replacement.)

Reply:
It is a glitch in the examples. I will rename it a `body_source`.